src/Proto/ComputationToComputationContainer/computation_to_computation.proto

```diff
@@ -37,9 +44,16 @@ message Share {
 */
 message Shares {
     message Share{
-        string value = 1;
+        oneof value{
+            int32 num = 4;
+            int64 num64 = 5;
+            bytes byte = 6;
+            float f = 7;
+            double d = 8;
+            bool flag = 9;
+        }
         Address address_id = 2;
     }
-    repeated Share share_list = 1; // list of share
+    repeated Share share_list= 1;
```

Comment:
```suggestion
    repeated Share share_list = 1;
```

Reply:
@come2ry Thank you for your review.  
It is fixed.  Please check these.